auto_backup/data/backup_data.yml

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
You can see in the comment above (https://github.com/odoo/odoo/issues/6287#issuecomment-240459221) the removal part. About deprecation, I don't remember where it was said, but it has been stated several times.

I see no point in discussing this: if the author of the PR agrees to change the test, perfect. If not, better YML test than nothing. In v10 we will need to change them. It's not a blocking point.